packages/ubuntu_bootstrap/lib/pages/install/install_model.dart

```diff
@@ -60,12 +61,14 @@ class InstallationEvent {
 /// View model for [InstallPage].
 class InstallModel extends SafeChangeNotifier {
   /// Creates an instance with the given client.
-  InstallModel(this._client, this._journal, this._product, this._session);
+  InstallModel(
+      this._client, this._journal, this._product, this._session, this._config);
```

Comment:
Add a trailing comma after `this._config` and reformat

Reply:
Yes, yes... sorry. That answer was for the "lgtm" part. Maybe I should put this as a draft until I add that test. Sorry.